source/configuration_manual/mail_crypt_plugin.rst

```diff
@@ -367,13 +367,14 @@ fix this later.
 
 FS driver syntax::
 
-  crypt:[algo=<s>:][set_prefix=<n>:][private_key_path=/path:][public_key_path=/path:][password=password:]<parent fs>``
+  crypt:[maybe][algo=<s>:][set_prefix=<n>:][private_key_path=/path:][public_key_path=/path:][password=password:]<parent fs>``
 
 where:
 
 ===================== ========================================================
 Key                   Value
 ===================== ========================================================
+``maybe``             Allow missing encryption keys. Added in v2.4;v3.0.
```

Comment:
Seems like this should have proper versionadded.

Also a note to upgrading page since this changes existing behavior.